clickhouse/columns/date.cpp

```diff
@@ -9,15 +9,15 @@ ColumnDate::ColumnDate()
 }
 
 void ColumnDate::Append(const std::time_t& value) {
-    data_->Append(static_cast<uint16_t>(value / std::time_t(86400)));
+    data_->Append(static_cast<uint16_t>(value / 86400));
 }
 
 void ColumnDate::Clear() {
     data_->Clear();
 }
 
 std::time_t ColumnDate::At(size_t n) const {
-    return data_->At(n) * std::time_t(86400);
+    return data_->At(n) * 86400;
```

Comment:
This is wrong and break tests.

PS. The initial code is fundamentally wrong BTW.